### PR TITLE
feat: add default after pipelines in result middleware

### PR DESCRIPTION
**Pull request by JonathanGriffe**

### CODE REVIEW

File: tests/middleware/test_retries.py [hunk 1]
```diff
@@ -1,33 +0,0 @@
-import remoulade
```
Comment (SimonDosda) [inline tests/middleware/test_retries.py:1]: why was this removed?

**Review by SimonDosda — CHANGES_REQUESTED** [review]
ok for me, you can add the changelog

File: tests/middleware/test_retries.py (hunk 1 above)
Comment (JonathanGriffe) [reply tests/middleware/test_retries.py:1]: It tested a feature that was not useful, using default_before that was deleted.
